### PR TITLE
Entity profile front end

### PR DESCRIPTION
**Pull request by musangowope**
Demo:
https://www.loom.com/share/3adbd045e9144599b6ca26f9ba8e5a95

Added entity profiles to the following pages:
- court detail
- author detail
- taxonomy detail

### CODE REVIEW

File: peachjam/templates/peachjam/court_detail.html [hunk 1]
```diff
@@ -7,13 +7,67 @@
...
+                    <strong>{% trans 'Physical address:' %}</strong>
+                  </div>
+                  <div>{{ entity_profile.address }}</div>
+                </div>
+              {% endif %}
+
+
+              {% if entity_profile.website_url %}
+                <div class="col-12 col-sm-6">
+                  <div><strong>{% trans 'Visit website:' %}</strong></div>
```
Comment (longhotsummer) [inline peachjam/templates/peachjam/court_detail.html:62]: I noticed that `:` crept in for advanced search. We don't use them anywhere else AFAIK, I think we shouldn't start introducing them. The bold and positioning is sufficient.

Comment (musangowope) [reply peachjam/templates/peachjam/court_detail.html]: removed

**Comment by musangowope** [comment]
@longhotsummer 
is the reason why we wrap the `court_detail.html` in a `blocktrans`
https://github.com/laws-africa/peachjam/blob/fc04f25265b062f8bc41e14b6cac2cab29799465/peachjam/templates/peachjam/court_detail.html#L28
is the format for the a court title different depending on the language?

File: peachjam/templates/peachjam/_entity_profile.html [hunk 2]
```diff
@@ -0,0 +1,50 @@
+{% load i18n %}
+
+<div class="entity-profile">
+  <div class="entity-profile__background-photo">
+    {% if entity_profile.background_photo %}
+      <img src="{{ entity_profile.background_photo.url }}"
+           alt="Court background image"
```
Comment (longhotsummer) [inline peachjam/templates/peachjam/_entity_profile.html:7]:
```suggestion
           alt="background image"
```

**Comment by longhotsummer** [comment]
@musangowope this looks awesome.

My only concern is how/when we repeat the title above the document list.

* author: "Documents by XXX" -- makes sense
* taxonomy: "Documents by Air Pollution" -- doesn't make sense
* court AND year: "Constitution Court - 2022" in title, then "Documents by Constitution Court" -- no year :/

Is it simpler to just effectively duplicate the heading if there's an entity profile block, and drop the "Documents by"? For the court and year case, the entity title for the court wouldn't have the year, but then the list above the documents would.

**Comment by musangowope** [comment]
Changes made:
- Authors still use Documents by
- taxonomy detail and court detail use the same title as their entity profile blocks and drop (Documents by)

**Pull request merged**